appveyor.yml

```diff
@@ -1,3 +1,24 @@
+# The config expects the following environment variables to be set:
+#  - "GN_CONFIG" Build type. One of {'debug', 'testing', 'release'}.
+#  - "GN_EXTRA_ARGS" Additional gn arguments for a build config,
+#      e.g. 'target_cpu="x86"' to build for a 32bit platform.
+#      https://gn.googlesource.com/gn/+/master/docs/reference.md#target_cpu
+#  - "ELECTRON_RELEASE" Define it to upload binaries on success.
```

Comment:
be specific about `define it`. should it be set to `1` or `"true"` or `"yes"`?

Reply:
Any value that would cause powershell's `Test-Path Env:\ELECTRON_RELEASE` to be evaluated to `True` works.

Reply:
Cool, pick one and document it here please.

Reply:
@jkleinsc do we usually set `ELECTRON_RELEASE` to `1`?

Reply:
@alexeykuzmin Yes we normally set `ELECTRON_RELEASE` to `1` and `UPLOAD_TO_S3` to `1` when we need them defined. 

In some cases we explicitly check for the `1` value: https://github.com/electron/electron/blob/master/.circleci/config.yml#L218